plugin-build/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ tasks.withType<KotlinCompile>().configureEach {
     kotlinOptions {
         jvmTarget = "1.8"
         freeCompilerArgs = listOf("-Xopt-in=kotlin.RequiresOptIn")
+        languageVersion = "1.3"
```

Comment:
I'm unsure we need this.

Reply:
its not required, its just better compatibility and fix https://github.com/getsentry/sentry-android-gradle-plugin/issues/104
we're only using the `capitalize` from Kotlin 1.4, not a biggie

Reply:
> its just better compatibility and fix #104

It doesn't actually Fix #104. To fix that you would have to downgrade the Kotlin version here https://github.com/getsentry/sentry-android-gradle-plugin/blob/6d92620153554a4c581b38b56ef4c805cd921ef0/buildSrc/src/main/java/Dependencies.kt#L4

Reply:
well it does if you've written Kotlin 1.3 compatible code, it could be run using Kotlin >= 1.4
To be sure you've written 1.3 compatible code, `languageVersion` is the way to go afaik

Reply:
Got it 👍 Let's go for it. We're not making extended usage of Kotlin 1.4 APIs so that's not a major deal.